backend/backend/core/tools.py

```diff
@@ -1,4 +1,5 @@
 import os
+import re
```

Comment:
Importing re is not necessary as os already has the functionality to handle regular expressions.